### DISCUSSION TITLE
Integrate RINEX to the Georust community?

### DISCUSSION THREAD

**Discussion by gwbres** [georust/geo#926] [Q&A] [answered]
Hello,

Sorry but for some reason I cannot log into the Discord channel, so I will contact you here.

We are 2 people working on a [RINEX](https://github.com/gwbres/rinex) and we're in the process of reaching out to communities to see if there is any interest in integrating our crate to a global project. The main objective would be to reach a broader audience, have more people involved in the maintenance, have the code naturally become more stable as time goes by.

To this day, the library can process in most common RINEX formats. We're adding RINEX data production these days, and also RINEX calculations methods. We have two binaries that replace the existing (but no longer maintained?) `CRX2RNX` and `RNX2CRX` official applications. Also we have a command line application that will eventually become about as capable as `teqc`

So the thing is, RINEX is certainly used by people involved in Geodesics, as it was designed as an open data exchange format. But it is used in other fields which are not related to Geodesics, like precise timing applications for instance.

Any thoughts on such a suggestion ?

**Comment by lnicola** [comment, marked as answer]
This came up on Discord in the context of another crate, but @urschrei said:

> I think we’d be delighted to have you on board. The main advantage to you is (as you say) more interested people, and more ongoing maintenance, as we try to keep up with MSRV and crate versions changing, and PR and issue admin – it can vary between crates, depending on how much use they get and what people are interested in, but we have a good track record (core geo crates date back almost to Rust 1.0, and we’ve been (co)-maintaining various ecosystem crates for about five years). The way we usually proceed is that you retain admin and crate publishing rights, along with georust “core” (a slowly-growing group of committers to one or more geo crates). I think the only other condition is adoption of the georust CoC (which is the Rust CoC). 

Still, I think it would apply here just as well.

PS: on a procedural note, https://github.com/georust/meta would have probably been a better place to ask, as `geo` is just one of the GeoRust crates.

**Reply by gwbres** [reply]
Hello @lnicola

>PS: on a procedural note, https://github.com/georust/meta would have probably been a better place to ask, as geo is just one of the GeoRust crates.

Sorry I was not aware of that. I just opened an issue in the [proper portal](https://github.com/georust/meta/issues/31)
